src/utils.js

```diff
@@ -27,21 +28,21 @@ const indexOf = function(list, elem) {
 /**
  * Return whether an element is contained in a list
  */
-const contains = function(list, elem) {
+const contains = function<T>(list: Array<T>, elem: T) {
     return indexOf(list, elem) !== -1;
 };
 
 /**
  * Provide a default value if a setting is undefined
  */
-const deflt = function(setting, defaultIfUndefined) {
+const deflt = function<T>(setting: T | void, defaultIfUndefined: T): * {
```

Comment:
Unfortunately, I couldn't use `T` as the output type itself as I ran into https://github.com/facebook/flow/issues/5022.

https://flow.org/try/#0MYewdgzgLgBAJgUwGYBsoEYYF4ZIK5jBQCW4APACoB8AFBAlCWAOYBcMFMAPjAG4jE4AGnjIAhnjQBJJAFUwiJMTAI47CgEp2AKhgBvAFAxjMAE4M8psDHqNlzbFhwFFy1TAD8opBOlyFyG5wMOy2TMwA3AYAvlGgkLCKaABM2LgERKRglLRh9urcfALC3r5QMvKuKmocWhz6RibmUJbWeSyOzgFK1Z6lkuX+Ve6hDOFRsQYG8dAwxBAAQmKm6OwARiAgKAhi1jhJGDQugdUiPij0GnHgs-NLpsnrm9u7aQfJR91BZ2IXCFcGIA

Reply:
Done.